src/OrleansAzureUtils/Hosting/AzureConfigUtils.cs

```diff
@@ -125,7 +127,8 @@ private static IEnumerable<DirectoryInfo> FindAppDirectoryLocations()
                 if (appRootPath != null)
                     locations.Add(new DirectoryInfo(appRootPath));
             });
-
+#if !NETSTANDARD_TODO
+            //System.Web namespace is deprecated in netstandard, need to find a alternative way to get {ServerRoot}
```

Comment:
I'm fine with leaving this as TODO for now, but I also believe we should simply drop the unavailable bits.